feat: introduction section updates (#255)
worldcoin/world-id-docs: src/pages/world-id.mdx

# World ID {{ className: 'uppercase text-5xl' }}
# World ID {{ className: 'text-5xl' }}

The protocol to bring global proof of personhood to the internet. {{ className: 'text-2xl' }}

{/* <hr className="my-10" /> */}

{/* <Note>

> 	You can find the Protocol Whitepaper at [whitepaper.worldcoin.org](https://whitepaper.worldcoin.org).

Reviewer: Let's still link the whitepaper somewhere in this page?

> World ID is a digital passport that lets a user prove they are a unique and real person while remaining anonymous. This happens through [zero-knowledge proofs](/further-reading/zero-knowledge-proofs) (ZKP) and other privacy-preserving cryptographic mechanisms.

Reviewer 2: World ID is a digital identity solution enabling users to prove their uniqueness and humanity anonymously via zero-knowledge proofs and advanced privacy-preserving cryptography.

Suggestions for making it more concise. 
Also, I prefer digital identity since it's more familiar and broad. Digital Passport sounds like we are offering some type of e-passport solution.

> World ID can be easily integrated into both cloud and on chain apps. Users, with a valid World ID, can anonymously verify in seconds they are a unique human through a World ID compatible wallet, like [World App](https://worldcoin.org/download).

Reviewer 2: ```suggestion
World ID can be easily integrated into both cloud and on chain apps. Users, with a valid World ID, can anonymously verify in seconds that they are a unique human through a World ID compatible wallet, like [World App](https://worldcoin.org/download).
```

> Developers can easily integrate World ID into their apps, whether in the cloud or on-chain. World ID holders can easily and anonymously verify they are a unique human through a compatible identity wallet, like [World App](https://worldcoin.org/download).

Reviewer 2: ```suggestion
World ID can be easily integrated into both cloud and on chain apps. Users, with a valid World ID, can anonymously verify in seconds they are a unique human through a World ID compatible wallet, like [World App](https://worldcoin.org/download).
```
